utils/model_uploader/update_models_upload_history_md.py

```diff
@@ -0,0 +1,246 @@
+        "Upload Time": uploader_time if uploader_time is not None else "-",
+        "Model ID": model_id,
+        "Model Version": model_version,
+        "Model Format": model_format,
+        "Embedding Dimension": str(embedding_dimension)
+        if embedding_dimension is not None
+        else "Default",
+        "Pooling Mode": pooling_mode if pooling_mode is not None else "Default",
+    }
+    return model_obj
+
+
+def update_model_json_file(
+    model_id: str,
+    model_version: str,
+    tracing_format: str,
+    embedding_dimension: Optional[int] = None,
+    pooling_mode: Optional[str] = None,
+    model_uploader: Optional[str] = None,
+    uploader_time: Optional[str] = None,
+) -> None:
+    """
+    Update supported_models.json
+
+    :param model_id: Model ID of the pretrained model
+    :type model_id: string
+    :param model_version: Version of the pretrained model for registration
+    :type model_version: string
+    :param tracing_format: Tracing format ("TORCH_SCRIPT", "ONNX", or "BOTH")
+    :type tracing_format: string
+    :param embedding_dimension: Embedding dimension input
+    :type embedding_dimension: int
+    :param pooling_mode: Pooling mode input ("CLS", "MEAN", "MAX", "MEAN_SQRT_LEN" or None)
+    :type pooling_mode: string
+    """
+    models = []
+    if os.path.exists(MODEL_JSON_FILEPATH):
+        with open(MODEL_JSON_FILEPATH, "r") as f:
+            models = json.load(f)
+    else:
+        os.makedirs(DIRNAME)
+
+    if tracing_format == TORCH_SCRIPT_FORMAT or tracing_format == BOTH_FORMAT:
+        model_obj = create_model_json_obj(
+            model_id,
+            model_version,
+            TORCH_SCRIPT_FORMAT,
+            embedding_dimension,
+            pooling_mode,
+            model_uploader,
+            uploader_time,
+        )
+        models.append(model_obj)
+
+    if tracing_format == ONNX_FORMAT or tracing_format == BOTH_FORMAT:
+        model_obj = create_model_json_obj(
+            model_id,
+            model_version,
+            ONNX_FORMAT,
+            embedding_dimension,
+            pooling_mode,
+            model_uploader,
+            uploader_time,
+        )
+        models.append(model_obj)
+
+    models = [dict(t) for t in {tuple(m.items()) for m in models}]
+    models = sorted(
+        models,
+        key=lambda d: (
+            d["Upload Time"],
+            d["Model ID"],
+            d["Model Version"],
+            d["Model Format"],
+        ),
+    )
+    with open(MODEL_JSON_FILEPATH, "w") as f:
+        json.dump(models, f, indent=4)
+
+
+def update_md_file():
+    """
+    Update MODEL_UPLOAD_HISTORY.md
+    """
+    models = []
+    if os.path.exists(MODEL_JSON_FILEPATH):
+        with open(MODEL_JSON_FILEPATH, "r") as f:
+            models = json.load(f)
+    models = sorted(
+        models,
+        key=lambda d: (
+            d["Upload Time"],
+            d["Model ID"],
+            d["Model Version"],
```

Comment:
can we keep `model version` before `model id`?

Reply:
Yes!